src/apps/activities/domain/activity_history.py

```diff
@@ -0,0 +1,51 @@
+import datetime
+import uuid
+
+from apps.shared.domain import InternalModel, PublicModel
+
+__all__ = [
+    "ActivityHistory",
+    "ActivityHistoryChange",
+    "PublicActivityHistoryChange",
+]
+
+
+class ActivityHistory(InternalModel):
+    id: int
+    applet_id: str
+    id_version: str
+    guid: uuid.UUID
+    name: str
+    description: dict
+    splash_screen: str
+    image: str
+    show_all_at_once: bool
+    is_skippable: bool
+    is_reviewable: bool
+    response_is_editable: bool
+    ordering: int
+    created_at: datetime.datetime
+
+
+class ActivityHistoryChange(InternalModel):
+    name: str | None
+    description: str | None
+    splash_screen: str | None
+    image: str | None
+    show_all_at_once: str | None
+    is_skippable: str | None
+    is_reviewable: str | None
+    response_is_editable: str | None
+    ordering: str | None
```

Comment:
1. if they are changed, then I will add text to them like, `name is changed from A to B`, or `ordering is updated to 8` if there is no change I leave it as `None`
2. the change text is provided by me
3. How you mean enumerates, can I create enumeration for range 1...1000 dynamically, ordering will depend on user input count
4. Images is not stored in our end, it will just provide path to CDN, and again this is for generating text for changes

Reply:
1, 2. Could you add more description here? It is not clear what this data model is responsible for.
3. I thought that ordering had a limited number of cases. Skip then
4. Maybe then we also should add more comments to describe. Or rename it to `image_path`

Reply:
Also, I think we should provide default values for all optional fields. Then all static type checker will not raise the error without plugins.
For example, this is my pyright output
![Alacritty 2023-01-24](https://user-images.githubusercontent.com/117179957/214289910-31c351d5-d612-466d-bc05-383c6f249111.png)